src/main/java/com/couchbase/lite/support/CouchbaseLiteHttpClientFactory.java

```diff
@@ -32,7 +32,7 @@
 import okhttp3.OkHttpClient;
 
 public class CouchbaseLiteHttpClientFactory implements HttpClientFactory {
-
+    private OkHttpClient client;
```

Comment:
Following is my personal preference. I understand that your solution is effective with minimum changes. For 1.3 release, this is good enough. But I would like to refactor after we release 1.3.

- In general, Factory is to create a target instance. If holding instance in a class, I prefer different name rather than Factory.
- By holding one instance in a Factory, it guarantees one instance per a factory. But this does not guarantees one instance per a replicator or a database. I prefer holding OkHttpClient instance in a replicator or a database.